### ISSUE TITLE
Unable to start Crafter Profile Admin due ClassDefFoundError: org.apache.http.client.HttpClient

### ISSUE THREAD

**Issue by cortiz** [craftercms/craftercms#4673]
## Describe the bug
I'm unable to start Crafter Profile Admin application due a 
`ClassDefFoundError: org.apache.http.client.HttpClient` error

## To Reproduce
1. Download Crafter CMS from the `[official website ](https://downloads.craftercms.org/3.1.13/social/crafter-cms-delivery-3.1.13.tar.gz)`
2. Start Crafter using the ./bin/startup.sh script'
3. Tail the logs
4. See 
```
n with name 'crafter.profileRestTemplate' defined in class path resource [crafter/profile/client-context.xml]: Instantiation of bean failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.craftercms.commons.rest.RestTemplate]: Constructor threw exception; nested exception is java.lang.NoClassDefFoundError: org.apache.http.client.HttpClient
2021-04-26_08:40:02.295 [localhost-startStop-1] ERROR org.springframework.web.context.ContextLoader - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'crafter.profileRestTemplate' defined in class path
resource [crafter/profile/client-context.xml]: Instantiation of bean failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.craftercms.commons.rest.RestTemplate]: Constructor threw exception; nested exception is java.lang.NoClassDefFoundError: org.apache.http.client.HttpClient
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateBean(AbstractAutowireCapableBeanFactory.java:1158) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1102) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:511) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:756) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867) ~[spring-context-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542) ~[spring-context-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443) ~[spring-web-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325) [spring-web-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107) [spring-web-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4701) [catalina.jar:8.5.53]
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5167) [catalina.jar:8.5.53]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:8.5.53]
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743) [catalina.jar:8.5.53]
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719) [catalina.jar:8.5.53]
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705) [catalina.jar:8.5.53]
        at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:970) [catalina.jar:8.5.53]
        at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1841) [catalina.jar:8.5.53]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_292]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_292]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_292]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_292]
        at java.lang.Thread.run(Thread.java:823) [?:1.8.0_292]
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.craftercms.commons.rest.RestTemplate]: Constructor threw exception; nested exception is java.lang.NoClassDefFoundError: org.apache.http.client.HttpClient
        at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:157) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:89) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateBean(AbstractAutowireCapableBeanFactory.java:1150) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        ... 26 more
Caused by: java.lang.NoClassDefFoundError: org.apache.http.client.HttpClient
        at org.craftercms.commons.rest.RestTemplate.<init>(RestTemplate.java:43) ~[crafter-commons-utilities-3.1.13.jar:3.1.13]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_292]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_292]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_292]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_292]
        at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:145) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:89) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateBean(AbstractAutowireCapableBeanFactory.java:1150) ~[spring-beans-4.3.29.RELEASE.jar:4.3.29.RELEASE]
        ... 26 more
Caused by: java.lang.ClassNotFoundException: org.apache.http.client.HttpClient
        at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1358) ~[catalina.jar:8.5.53]
        at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1180) ~[catalina.jar:8.5.53]
        at org.craftercms.commons.rest.RestTemplate.<init>(RestTemplate.java:43) ~[crafter-commons-utilities-3.1.13.jar:3.1.13]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_292]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_292]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_292]
```

## Expected behavior
Crafter Profile Admin should start 

## Screenshots
{{If applicable, add screenshots to help explain your problem.}}

## Logs
https://gist.github.com/cortiz/e3292203040eb958f0226d7d5b6508c8

### Version
Crafter 3.1.13
Java 1.8.0_292-b10 Eclipse OpenJ9

### OS
Linux and Docker Container

### Browser
N/A

**Comment by cortiz** [comment]
Looks like it's fixed on https://github.com/craftercms/profile/commit/d2a01b9323b0037435c0345a2d4bd25bfb5fa0ba

**Comment by cortiz** [comment]
Duplicate of 4611 Just not released to 3.1.x
